src/ted2zim/scraper.py

```diff
@@ -1020,30 +1021,33 @@ def upload_to_cache(self, key, object_path, encoder_version):
         return True
 
     def remove_failed_topics_and_check_extraction(self, failed_topics):
-        """removes failed topics from topics list and raises error if scraper cannot continue"""
+        """removes failed topics from topics list and raises error if scraper cannot
+        continue"""
 
         for topic in failed_topics:
             self.topics.remove(topic)
         if not self.topics:
             if self.source_languages:
                 raise ValueError(
-                    "No videos found for any topic in the language(s) requested. Check topic(s) and/or language code supplied to --languages"
+                    "No videos found for any topic in the language(s) requested. Check "
+                    "topic(s) and/or language code supplied to --languages"
                 )
             raise ValueError("Wrong topic(s) were supplied. No videos found")
 
     def run(self):
         logger.info(
             f"Starting scraper with:\n"
             f"  langs: {', '.join(self.source_languages)}\n"
-            f"  subtitles : {', '.join(self.subtitles_setting) if isinstance(self.subtitles_setting, list) else self.subtitles_setting}\n"
+            f"  subtitles : { ', '.join(self.subtitles_setting) if isinstance(self.subtitles_setting, list) else self.subtitles_setting}\n"  # noqa: E501
```

Comment:
Why a space at begining of expr in f-string?

Reply:
Well It's unbalanced so it's incorrect anyway. Our convention is to not use spaces here (see line above) so I think it should be fixed.

Reply:
Oh, we weren't speaking about the same space ; this one is weird, you are right